src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -13,6 +13,7 @@
     "test:config": "if [ ! -e 'src/unit-test-configuration.ts' ]; then cp 'src/unit-test-configuration.ts.sample' 'src/unit-test-configuration.ts'; fi",
     "e2e": "npm run env_build && ng e2e",
     "e2e:dev": "npm run env_build && ng e2e --dev-server-target",
+    "e2e:ci": "npm run env_build && ng e2e -c ci",
```

Comment:
I think we can use `e2e:dev` and get the same result.
```suggestion
```

Reply:
> I think we can use `e2e:dev` and get the same result.

@tspmelo I already realized that, but I think that "e2e:dev" is a misleading name as it seems
to be only for ```dev``` purposes, meanwhile ```ci``` clearly indicates that is related to CI tasks.
Should we replace e2e:dev with e2e:ci and drop e2e:dev?

Reply:
I'm ok with changing the name of `dev` to `ci`

Reply:
I'm fine with it, too :)